local/compose/create.go

```diff
@@ -388,36 +388,91 @@ func buildContainerMountOptions(p types.Project, s types.ServiceConfig, img moby
 	}
 	if img.ContainerConfig != nil {
 		for k := range img.ContainerConfig.Volumes {
-			mount, err := buildMount(p, types.ServiceVolumeConfig{
+			bindMount, err := buildMount(p, types.ServiceVolumeConfig{
```

Comment:
why rename var as `bindMount`? this slice also contain volume mounts, not just bind mounts

Reply:
I just renamed because `mount` is the name of an imported package. So my IDE is complaining about shadowing. I'll get a better name